pages/base_page.py

```diff
@@ -10,11 +10,12 @@ def __init__(self, driver):
     def go_to(self):
         self.driver.get(self.PAGE_URL)
 
-    def wait_for_id(self, id):
+    def get_clickable_element(self, *selector):
```

Comment:
This is a much better name - good job!
What happens for elements like images that don't hyperlink anywhere? Do they still register as clickable?